spyder/plugins/ipythonconsole/widgets/debugging.py

```diff
@@ -431,6 +431,20 @@ def set_pdb_state(self, pdb_state):
         if pdb_state is not None and isinstance(pdb_state, dict):
             self.refresh_from_pdb(pdb_state)
 
+    def pdb_out(self, text):
+        """Show pdb out."""
```

Comment:
```suggestion
    def show_pdb_output(self, text):
        """Show Pdb output."""
```

Let's rename this method to make clearer what it does.